app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt

```diff
@@ -754,17 +762,93 @@ fun Week(
         pageCount = { days.size }
     )
     val currentPage = pagerState.currentPage
+    var selectedFilter by remember { mutableStateOf(TaskFilter.Daily) }
+    val filteredTasks = filterTasks(tasks, selectedFilter)
```

Comment:
Split params. Then this block of code will be of multiple lines, separate it by a single line (top/bot).